### DISCUSSION TITLE
URL in grid

### DISCUSSION THREAD

**Discussion by legaljustus** [Budibase/budibase#1426] [Help]
Hi! I have a postgresql query where I store url's to documents as CharVar. I want to add a download link to my grid based on this query. I tried the following which has worked for me on other platforms:

`CONCAT('<a href=\"\" onclick=(\"' , {{ insert url column here }} , '\")>Click</a>') `

but unfortunately it just displays as a text. It also doesn't work if I try it without the onclick.

Is there a way to do this? 

I see a detail url option in the configuration of the grid but can't figure out how to use it.

**Comment by mjashanks** [comment]
Hi There. It would probably be much easier to use a `Link` component:

![image](https://user-images.githubusercontent.com/3524181/116857523-dc389e00-abf4-11eb-841a-fc51d88c8c0b.png)

If you need to do it with as above, with an `onclick`, you should be able to put your text inside an `Embed` component.
